packages/sui-bundler/webpack.config.dev.js

```diff
@@ -44,18 +45,7 @@ let webpackConfig = {
   ],
   module: {
     rules: [
-      {
-        test: /\.jsx?$/,
-        include: /src/,
```

Comment:
isn't this include needed?

Reply:
It works. And checkout the rest of the configs. No trace of it.
Will recheck anyways.